tests/test_cli.py

```diff
@@ -332,3 +333,88 @@ def _validate_intermediate_output(*args):
         server_b_published_output_filename,
     ):
         assert json.load(open(filename)) == [3, 2, 1]
+
+
+def test_verify1_ignores_bad_payloads_in_batch(tmp_path, shared_seed):
+    key_a = _keygen()
+    key_b = _keygen()
+    key_c = _keygen()
+
+    batch_id = "test"
+    n_data = 3
+    base_args = ["--n-data", n_data, "--batch-id", batch_id]
+
+    runner = CliRunner()
+
+    # create dependencies for encode function
+    data_path = tmp_path / "data.json"
+    with open(data_path, "w") as f:
+        f.write(json.dumps([1, 0, 1]))
+    ignore = tmp_path / "ignore"
+    ignore.mkdir()
```

Comment:
Maybe I'm being thick, but where is the bad data being generated here? Can you put a comment in this test around that section to make it more clear where that's happening?

Reply:
The bad data is being generated by using two different keys for server A:

    encode(key_a, key_c, out_0)
    encode(key_b, key_c, out_1)

When decrypting using `key_a`, the payload encoded using `key_b` will be invalid.

Reply:
Gotcha, makes sense. A comment that you are deliberately introducing an error here would make this much more clear.